psutil/psutil.go

```diff
@@ -115,16 +126,34 @@ func (p *Psutil) GetMetricTypes(_ plugin.ConfigType) ([]plugin.MetricType, error
 		return nil, err
 	}
 	mts = append(mts, mts_...)
+	mts = append(mts, getDiskUsageMetricTypes()...)
 
 	return mts, nil
 }
 
 //GetConfigPolicy returns a ConfigPolicy
 func (p *Psutil) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
 	c := cpolicy.New()
+	config := cpolicy.NewPolicyNode()
+	r, err := cpolicy.NewStringRule("mount_points", false)
+
+	if err != nil {
+		return nil, err
+	}
+	r.Description = "Mountpoints"
+	config.Add(r)
+	c.Add([]string{""}, config)
```

Comment:
I haven't noticed it's not added as required, so snap will validate this config successfully when given in task, however it is still not valid to add it at root node:
```
lmroz@lmroz-DEV:~/.gvm/pkgsets/go1.7.4/global/src/github.com/intelsdi-x/snap/build/linux/x86_64$ ./snaptel metric get -m "/intel/psutil/disk/*/used"
NAMESPACE                                VERSION         UNIT    LAST ADVERTISED TIME                    DESCRIPTION
/intel/psutil/disk/[mount_point]/used    10                      Fri, 31 Mar 2017 13:35:55 CEST          

  Dynamic elements of namespace: /intel/psutil/disk/[mount_point]/used

       NAME              DESCRIPTION
       mount_point       Mount Point

  Rules for collecting /intel/psutil/disk/[mount_point]/used:

       NAME      TYPE    DEFAULT         REQUIRED        MINIMUM         MAXIMUM
lmroz@lmroz-DEV:~/.gvm/pkgsets/go1.7.4/global/src/github.com/intelsdi-x/snap/build/linux/x86_64$ ./snaptel metric get -m "/intel/scaleio/*/unusedCapacityInKb"
NAMESPACE                                                VERSION         UNIT    LAST ADVERTISED TIME                    DESCRIPTION
/intel/scaleio/[StoragePool]/unusedCapacityInKb          2                       Fri, 31 Mar 2017 13:30:17 CEST          

  Dynamic elements of namespace: /intel/scaleio/[StoragePool]/unusedCapacityInKb

       NAME              DESCRIPTION
       StoragePool       The specific storage pool ID to collect from

  Rules for collecting /intel/scaleio/[StoragePool]/unusedCapacityInKb:

       NAME              TYPE            DEFAULT         REQUIRED        MINIMUM         MAXIMUM
       gateway           string                          true                            
       username          string                          true                            
       password          string                          true                            
```
Perhaps you see, that configuration options for this metric cannot be listed. I do not know if there are any other consequences of adding it at root node.